runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -5922,6 +5922,9 @@ static int32_t J9THREAD_PROC samplerThreadProc(void * entryarg)
             javacoreData->numAotDataEntries,
             javacoreData->numJitHints,
             javacoreData->numJitProfiles);
+#if defined(OMR_GC_COMPRESSED_POINTERS)
+         TR_VerboseLog::writeLine(TR_Vlog_INFO, "\tAOT header compressedRefs shiftAmount=%u", vm->compressedPointersShift);
```

Comment:
The output needs to come from the AOT header stored in the SCC, not from the current VM. We already print the value for the current VM in some other part of the code.